### PR TITLE
Feature: CF create diagram from template

### LINKED ISSUE #441: Integrate conditional feature: CF_createDiagramFromTemplate
## Description

Integrate conditional feature to create a diagram from template.

Role: `CF_createDiagramFromTemplate`

add in `src/boot/vue-simple-acl.js`:
```js
setRule('create-template-diagram', (user) => !process.env.HAS_BACKEND || user.roles.includes('CF_createDiagramFromTemplate'));
```

modify impacted html component with:
```
:title="$acl.role('create-template-diagram')
            ? $t('CURRENT_TITLE')
            : $t('errors.permissionsDenied')"
:disable="!$acl.role('create-template-diagram')"
```

### PR DESCRIPTION
**Pull request by Zorin95670**
Resolved #441

### CODE REVIEW

File: src/boot/vue-simple-acl.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ const userStore = useUserStore();
 const userRoles = computed(() => ({ roles: userStore.roles }));
 const rules = () => defineAclRules((setRule) => {
   setRule('create-diagram', (user) => !process.env.HAS_BACKEND || user.roles.includes('CF_createDiagram'));
+  setRule('create-template-diagram', (user) => !process.env.HAS_BACKEND || user.roles.includes('CF_createDiagramFromTemplate'));
```
Comment (k-y0u) [inline src/boot/vue-simple-acl.js:10]: `create-template-diagram` -> `create-diagram-from-template`

**Review by k-y0u — CHANGES_REQUESTED** [review]

File: src/boot/vue-simple-acl.js (hunk 1 above)
Comment (k-y0u) [reply src/boot/vue-simple-acl.js:10]: why ?

Comment (Zorin95670) [reply src/boot/vue-simple-acl.js:10]: I do what the issue says and if I remember you was present when we write it :)

Comment (aboubakar14) [reply src/boot/vue-simple-acl.js:10]: I agree with  @Zorin95670, no need to add the from. it will be too long and its already clear without the from

Comment (k-y0u) [reply src/boot/vue-simple-acl.js:10]: The problem is it could be understand as "You can/can't create a new template diagram."...

Comment (Zorin95670) [reply src/boot/vue-simple-acl.js]: done

**Review by k-y0u — APPROVED** [review]

**Review by aboubakar14 — APPROVED** [review]

**Pull request merged**